SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -40,13 +41,33 @@ FORCEINLINE UClass* ResolveClass(FString& ClassPath)
 	return Class;
 }
 
+void USpatialClassInfoManager::QuitGame(UObject* WorldContext, UClass* Class)
```

Comment:
This is awesome. I'm wondering if the `USpatialClassInfoManager` is the correct location for this method though. Perhaps it would be better living in the `SpatialGameInstance`?

Reply:
Ended up inlining the logic as per Giray's comment.